compositions/aws-provider/backup-velero/backup.yaml

```diff
@@ -0,0 +1,254 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: backup.awsblueprints.io
+  labels:
+    awsblueprints.io/provider: aws
+    awsblueprints.io/environment: dev
+spec:
+  compositeTypeRef:
+    apiVersion: awsblueprints.io/v1alpha1
+    kind: XBackup
+  patchSets:
+    - name: common-fields
+      patches:
+        - fromFieldPath: spec.resourceConfig
+          toFieldPath: spec.resourceConfig
+          policy:
+            mergeOptions:
+              keepMapValues: true
+
+    # ToDo(haarchri): https://github.com/crossplane/crossplane/issues/3139
+    # - name: bucket-status
+    #   patches:
+    #     - type: ToCompositeFieldPath
+    #       fromFieldPath: status.bucketArn
+    #       toFieldPath: status.backupBucket.bucketArn
+    #       policy:
+    #         fromFieldPath: Optional
+    #     - fromFieldPath: status.bucketName
+    #       toFieldPath: status.backupBucket.bucketName
+    #       type: ToCompositeFieldPath
+    #       policy:
+    #         fromFieldPath: Optional
+
+    - name: chart-status
+      patches:
+        - type: ToCompositeFieldPath
+          fromFieldPath: status.atProvider
+          toFieldPath: status.backupInformations.release
+          policy:
+            fromFieldPath: Optional
+        - type: ToCompositeFieldPath
+          fromFieldPath: status.failed
+          toFieldPath: status.backupInformations.releaseFailed
+          policy:
+            fromFieldPath: Optional
+
+    - name: chart-parameters
```

Comment:
Probably a good idea to include image tags so that users can override Velero and AWS init container images with w/e they want to use.